### ISSUE TITLE
CMS, components, design system, sitewide - 508-defect-2 [KEYBOARD, AXE-CORE, SEMANTIC MARKUP]: Landmarks MUST have a unique role or role/label/title (i.e. accessible name) combination; main nav and sidebar nav conflict

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#7723]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

```diff
! Team & project: vsa-public-websites, AMA follow-on
```

Please see issue https://github.com/department-of-veterans-affairs/va.gov-team/issues/6368 — this is the same issue, but in a different project.

> **Please note:** This particular ticket is an issue with the CMS templates, and also exists throughout our React apps. The main navigation, side navigation, and breadcrumbs need distinct `aria-labels` to distinguish them for assistive technology. Ideally, this would also be fixed in our React apps.
> * main nav: aria-label="main"
> * side nav: aria-label="secondary"
> * breadcrumbs: aria-label="breadcrumb"

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Currently the VA.gov main nav, breadcrumbs, and the sidebar nav use the `<nav>` element. Each landmark **MUST** have a unique role or accessible name (i.e. role, label, title) combination. Sighted keyboard and screen reader users navigate pages by landmarks and need to be able to distinguish between them. If landmark roles are repeated on a page, then they need to be labelled uniquely.

1. After you request a decision review — https://preview-prod.vfs.va.gov/preview?nodeId=3008 | `<div id="vetnav" role="navigation">` and #va-detailpage-sidebar [strange, because I don't see a sidebar on this page]
1. Supplemental Claims — https://preview-prod.vfs.va.gov/preview?nodeId=3011 | same as above
1. Higher-Level Reviews — https://preview-prod.vfs.va.gov/preview?nodeId=3003 | same as above
1. Board Appeals — https://preview-prod.vfs.va.gov/preview?nodeId=3029 | same as above
1. Contested Claims — https://preview-prod.vfs.va.gov/preview?nodeId=3030 | same as above
1. Fiduciary claims — https://preview-prod.vfs.va.gov/preview?nodeId=3016 | same as above
1. Insurance claims — https://preview-prod.vfs.va.gov/preview?nodeId=3025 | same as above
1. FAQs — https://preview-prod.vfs.va.gov/preview?nodeId=3004 | same
1. Get help with review request — https://preview-prod.vfs.va.gov/preview?nodeId=3009 | same
1. Board hearings with a VLJ — https://preview-prod.vfs.va.gov/preview?nodeId=3010 | same

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a keyboard user, I want to navigate the page by landmarks and distinguish between each landmark.

## Environment

* Operating System: all
* Browser: all
* Screen reader: any
* Server destination: production

## Steps to Recreate

1. Enter any of the urls above  in browser (in the VA environment)
2. Start screenreading device of choice
3. Navigate the page by landmarks (see vendor guidance below)
4. Alternatively, scan the page with axe or axe-coconut to verify the issue

## Possible Fixes (optional)

There are three navigation items on this page:
- sitewide main vetnav megamenu
- breadcrumbs component
- sidebar nav

If a page includes more than one navigation landmark, each should have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable

```diff
! <!-- current -->
! <div id="vetnav" role="navigation">
! <nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs" id="va-breadcrumbs">
! <nav id="va-detailpage-sidebar" data-drupal-sidebar="true" class="va-drupal-sidebarnav usa-width-one-fourth va-sidebarnav">

<!-- possible fix -->
<nav id="vetnav"> <!-- ideally the main nav would be a nav element, thereby wouldn't need the aria-label="main" -->
<nav aria-label="breadcrumb" aria-live="polite" class="va-nav-breadcrumbs" id="va-breadcrumbs">
<nav aria-label="secondary" id="va-detailpage-sidebar" data-drupal-sidebar="true" class="va-drupal-sidebarnav usa-width-one-fourth va-sidebarnav">
```

## WCAG or Vendor Guidance (optional)

* [W3C ARIA Navigation Landmark Examples](https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/landmarks/navigation.html)
* [axe-core 3.4 - Landmarks must have a unique role or role/label/title (i.e. accessible name) combination](https://dequeuniversity.com/rules/axe/3.4/landmark-unique?application=AxeChrome)
* [Apple VoiceOver - Navigate webpages using landmarks](https://help.apple.com/voiceover/info/guide/10.11/en.lproj/vo35709.html)

**Comment by laflannery** [comment]
Duplicate issues as it's sitewide - closing in favor of #6368
